spec/controllers/applications_controller_spec.rb

```diff
@@ -59,4 +59,15 @@
       end
     end
   end
+
+  describe '#edit' do
+    it 'should redirect on lottery completed' do
+      VCR.use_cassette('applications_controller/edit/non_lease_up_application') do
+        get :edit, params: { id: non_lease_up_application_id }
```

Comment:
minor point, but it looks like this application (https://sfhousing--full.lightning.force.com/lightning/r/Application__c/a0o0P00000Fv20MQAR/view) is actually redirecting because it's an electronic app, and not because lottery is complete. Maybe just update the title? or use this one as an electronic app example, and add another example for a paper app that has lottery status complete (although it doesn't look like we have one ready to go in force_helpers.rb)